tests/expectations/exceptionWithInvalidCode.json

```diff
@@ -2,7 +2,8 @@
   "errors": [
     {
       "code": "600",
-      "title": "Some error occurred"
+      "title": "Some error occurred",
+      "detail": "foobar"
```

Comment:
Why is there everywhere now also a `detail`? Doesn't really match the file-name anymore now 😬